Adapter_oM/Versioning40.json

```diff
@@ -0,0 +1,9 @@
+{
+    "Property": {
+        "ToNew": {
+            "BH.oM.Adapter.ActionConfig.DiffConfig": "BH.oM.Adapter.ActionConfig.ComparisonConfig"
```

Comment:
I am not sure this is enough actually, because two changes happened:
- The property name changed (from `DiffConfig` to `ComparisonConfig`)
- The property type changed (from `DiffConfig` to `ComparisonConfig`, two different classes)

How do we deal with this change? Is adding a "Type" section in the Versioning here correct? @adecler

Reply:
If all the `DiffConfig` are now `ComparisonConfig` types, then yes. 
If it doesn't work for some reason of the combination of the two not being properly supported, you will have to do a custom converter for mow.